### ISSUE TITLE
Object missing content

### ISSUE THREAD

**Issue by CA10C3SAR** [zero-plusplus/vscode-autohotkey-debug#171]
I tried to reproduce the `error(?)`, follow:

```
x      := "a`nb`nc"
y      := {1: "d`ne`nf"}
obj    := {}
obj[x] := y

MsgBox, 0,MsgBox,  % obj["a`nb`nc"].1
return
```

Using the code above I cant see the value of the object:
![image](https://user-images.githubusercontent.com/6724006/143256249-d563dff2-152f-42da-a385-da5b2695c8a4.png)

Is it a bug?

**Comment by zero-plusplus** [comment]
This is a bug because SciTE4AutoHotkey does not have this issue.

It may take some time to fix this issue.

**Comment by zero-plusplus** [comment]
This issue has been fixed.

The release that includes the fix for this issue will be available as early as this month and as late as next month.

Please note that this is a large update and it will take some time to check.

Thanks for the report.

**Comment by zero-plusplus** [comment]
@CA10C3SAR 
I released `1.10.0` which includes a fix for this issue.

Thanks for the report.

**Comment by SAbboushi** [comment]
WOW!  Judging by all those email notifications this morning, you've been BUSY!!
